src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -107,6 +108,11 @@ internal data class StandaloneStoppedNode(
         ssh: SshConnection,
         threadDump: ThreadDump
     ) {
+        if (skipStartupWait) {
```

Comment:
We can reuse the existing terminology:
```suggestion
        if (!waitForUpgrades) {
```

Reply:
Applied this suggestion